libopenthread/platform/radio.c

```diff
@@ -0,0 +1,281 @@
+#include <net/ieee802154.h>
+#include <libtock-sync/net/ieee802154.h>
+#include <openthread-system.h>
+#include <openthread/platform/radio.h>
+#include <plat.h>
+#include <stdio.h>
+
+#define ACK_SIZE 3
+
+
+static uint8_t tx_mPsdu[OT_RADIO_FRAME_MAX_SIZE];
+static otRadioFrame transmitFrame = {
+  .mPsdu   = tx_mPsdu,
+  .mLength = OT_RADIO_FRAME_MAX_SIZE
+};
+
+// nrf52840 ACK on transmit is currently unimplemented. We fake this
+// for now by always saying the sent packet was ACKed by the receiver.
+static uint8_t ack_mPSdu[ACK_SIZE] = {0x02, 0x00, 0x00};
+static otRadioFrame ackFrame = {
+  .mPsdu   = ack_mPSdu,
+  .mLength = 3
+};
+
+void otPlatRadioGetIeeeEui64(otInstance *aInstance, uint8_t *aIeeeEui64) {
+  // TODO HARDCODED FOR NOW
+  OT_UNUSED_VARIABLE(aInstance);
+  aIeeeEui64[0] = 0xf4;
+  aIeeeEui64[1] = 0xce;
+  aIeeeEui64[2] = 0x36;
+  aIeeeEui64[3] = 0x67;
+  aIeeeEui64[4] = 0x13;
+  aIeeeEui64[5] = 0x12;
+  aIeeeEui64[6] = 0x3f;
+  aIeeeEui64[7] = 0xa6;
```

Comment:
Not dynamic?

Reply:
This change got lost in the rebase/merge. Updated now.